docs/pages/desktop-access/getting-started.mdx

```diff
@@ -138,36 +188,88 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
+dsacls "CN=TELEPORT,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G “$($TeleportLDAPGroupName):CCDC;cRLDistributionPoint;"
+# Gives Teleport the ability to write the certificateRevocationList property in the CDP/Teleport container.
+dsacls "CN=TELEPORT,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN " /I:T /G “$($TeleportLDAPGroupName):WP;certificateRevocationList;"
+# Gives Teleport the ability to create and delete certificationAuthority objects in the NTAuthCertificates container.
+dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($TeleportLDAPGroupName):CCDC;certificationAuthority;"
+# Gives Teleport the ability to write the cACertificate property in the NTAuthCertificates container.
+dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($TeleportLDAPGroupName):WP;cACertificate;"
 
-```text
-Computer Configuration > Policies > Windows Settings > Security Settings > Windows Firewall and Advanced Security
 ```
 
-Right click on `Inbound Rules` and select `New Rule...`. Under `Predefined`
-select `Remote Desktop`. Only select the rule for `User Mode (TCP-in)`. On the
-next screen, select `Allow the connection` and finish.
+## Step 4/5. Configure Teleport
+
+<Admonition
+  type="note"
+  title="Teleport CA"
+>
+  Prior to v8.0, the Teleport CA was not compatible with Windows logins. If
+  you're setting up Desktop Access in an existing cluster created before v8.0,
+  you must first perform a [CA rotation](../setup/operations/ca-rotation.mdx)
+  in order to resolve this.
+</Admonition>
+
+First, we need to enable Desktop Access in Teleport. To do this, add the
+following section in `teleport.yaml` on your Linux server:
+
+```yaml
+windows_desktop_service:
+  enabled: yes
+  # This is the address that windows_desktop_service will listen on.
+  listen_addr: "0.0.0.0:3028"
+  # (optional) This is the address that windows_desktop_service will advertise
+  # to the rest of Teleport for incoming connections. Only proxy_service should
+  # connect to windows_desktop_service, users connect to the proxy's web UI
+  # instead.
+  public_addr: "desktop-access.example.com:3028"
+  ldap:
+    # Address of the Domain Controller for LDAP connections. Usually, this
+    # address will use port 389, like: domain-controller.example.com:389.
+    addr:     '$LDAP_SERVER_ADDRESS'
+    # Active Directory domain name you are connecting to, like: domain-controller.example.com.
+    domain:   '$LDAP_DOMAIN_NAME'
+    # LDAP username for authentication. This will be the username you chose for the service 
+    # account User in step 1. This username must include the domain NetBIOS name.
+    #
+    # For example, if your domain is "example.com", the NetBIOS name for it is
+    # likely "EXAMPLE". When connecting as the "TLDAPSA" user, you should
```

Comment:
Let's try to stick to the convention of `svc-xxx` for service accounts, and recommend something like `svc-teleport` for this account name.